.agent/commands/final-review.md

```
@@ -1,18 +1,18 @@
 
 This command is the SoT for review taxonomy (status/priority) shared by:
 
-- `/review` (human-readable Japanese review output)
+- `/final-review` (human-readable Japanese review output)
 - `/review-cycle` (machine-readable `review.json` output)
 
 User-facing output remains in Japanese.
 
 ## Usage
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)